src/toolkits/style_transfer/style_transfer.cpp

```diff
@@ -463,9 +463,13 @@ gl_sframe style_transfer::predict(variant_type data,
         break;
       case flex_type_enum::VECTOR:
         style_idx = std::move(flex_style_idx.get<flex_vec>());
+        if(style_idx.size() == 0)
```

Comment:
Nit: use clang-format for consistent formatting, e.g. leave a space between `if` and opening parenthesis

For STL containers, prefer `.empty()` to `.size() == 0`

Reply:
Solid!